src/_includes/banner.html

```diff
@@ -1,9 +1,9 @@
 <div class="banner">
   <p class="banner__text">
-    Check out the
-    <a href="https://medium.com/dartlang/dart-3-2-c8de8fe1b91f">Dart 3.2 blog post</a>!
+    Dart 3.3 is out!
+    Get extension types, better JavaScript integration, and more.
```

Comment:
Not necessary for this PR if you prefer this wording, but I think it's best to avoid "get" if possible. It's not clear what "getting" extension types means.

Also would something like "improved JavaScript interop" provide more details to readers than "better JavaScript integration"?

Reply:
I agree, this was the "marketing approved" text but idk what that means and it's not technically great 😅 

Changing!